.github/workflows/kernel-test.yml

```diff
@@ -0,0 +1,70 @@
+name: Reusable test workflow
+
+on:
+  workflow_call:
+    inputs:
+      arch:
+        required: true
+        type: string
+      toolchain_full:
+        required: true
+        type: string
+      runs_on:
+        required: true
+        type: string
+      kernel:
+        required: true
+        type: string
+      test:
+        required: true
+        type: string
+      continue_on_error:
+        required: true
+        type: string
+      timeout_minutes:
+        required: true
+        type: number
+
+jobs:
+  test:
+    if: ${{ github.event_name != 'push' }}
```

Comment:
This condition doesn't really here, does it? It seems more like a property that the caller imposes, no?

Reply:
yeah, fair point, this should be moved up one level.